code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -281,7 +281,7 @@
 		time += M.rating
 	for(var/obj/item/stock_parts/capacitor/C in component_parts)
 		time += C.rating
-	recharge_delay /= time/2         //delay between recharges, double the usual time on lowest 50% less than usual on highest
+	recharge_delay = 30/(time/2)         //delay between recharges, double the usual time on lowest 50% less than usual on highest
```

Comment:
maybe `recharge_delay = initial(recharge_delay)/(time/2)`?

Reply:
That would work if we kept the initial recharge_delay at 30, but this patch also changes that to 20 which is the maximum quotient this formula will produce given 3 is the minimum value for time, and judging by the comment clearly the intended value.